meteor/server/databaseMigrationSystemSteps.ts

```diff
@@ -597,102 +735,102 @@ addMigrationSteps( '0.1.0', [
 		}]
 	},
 	{
-		id: 'Playout-gateway.casparcg0',
-		canBeRunAutomatically: false,
+		id: 'playout-gateway.caspar01',
+		canBeRunAutomatically: true,
 		validate: () => {
 			let device = PeripheralDevices.findOne({type: PeripheralDeviceAPI.DeviceType.PLAYOUT})
 			if (!device) return 'Playout-gateway not found'
 			let settings = device.settings || {devices: {}} as PlayoutDeviceSettings
 
-			let casparcg0 = settings.devices['casparcg0'] as PlayoutDeviceSettingsDeviceCasparCG
-			if (!casparcg0) return '"casparcg0" missing'
+			let caspar01 = settings.devices['caspar01'] as PlayoutDeviceSettingsDeviceCasparCG
+			if (!caspar01) return '"caspar01" missing'
 
 			// @ts-ignore
-			if (!casparcg0.options) casparcg0.options = {}
-			if (casparcg0.type !== PlayoutDeviceType.CASPARCG) return 'Type is not "CASPARCG"'
-			if (!casparcg0.options.host) return 'Host is not set'
-			if (!casparcg0.options.launcherHost) return 'Launcher host is not set'
+			if (!caspar01.options) caspar01.options = {}
+			if (caspar01.type !== PlayoutDeviceType.CASPARCG) return 'Type is not "CASPARCG"'
+			if (!caspar01.options.host) return 'Host is not set'
+			if (!caspar01.options.launcherHost) return 'Launcher host is not set'
 
 			return false
 		},
 		migrate: () => {
 			let device = PeripheralDevices.findOne({type: PeripheralDeviceAPI.DeviceType.PLAYOUT})
 			if (device) {
 				// Set some default values:
-				let casparcg0 = device.settings && device.settings.devices['casparcg0']
-				if (!casparcg0) {
-					logger.info(`Migration: Add PeripheralDevice.settings to ${device._id}: casparcg0`)
+				let caspar01 = device.settings && device.settings.devices['caspar01']
+				if (!caspar01) {
+					logger.info(`Migration: Add PeripheralDevice.settings to ${device._id}: caspar01`)
 					PeripheralDevices.update(device._id, {$set: {
-						'settings.devices.casparcg0': {
+						'settings.devices.caspar01': {
 							type: PlayoutDeviceType.CASPARCG,
 							options: {
-								host: '127.0.0.1',
-								port: 5250,
-								launcherHost: '127.0.0.1',
-								launcherPort: 8010, // todo: change this
+								host: '',
```

Comment:
@nytamin ref question on initiating values or if we could omit this in total (would need us to have a map for validation)